docs/components/mission/MissionPanel.tsx

```
@@ -0,0 +1,137 @@
+        </section>
+      )}
+
+      {/* Mission Phase Timeline */}
+      <section className="pt-6 border-t border-gray-700">
+        <PhaseTimeline phases={phases} />
+      </section>
+
+      {/* Auto-scroll trigger on tab change */}
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)